firefox_desktop/views/urlbar_events.view.lkml

```diff
@@ -1,223 +1,331 @@
+    sql: ${TABLE}.pref_fx_suggestions ;;
+    type: yesno
+    group_label: "User Preferences"
+    description: "Whether or not the checkbox for showing Firefox Suggestions is checked in about:preferences"
+  }
 
-dimension: num_quick_action_impressions {
-  hidden:  yes
-}
+  dimension: pref_sponsored_suggestions {
+    hidden: yes
+  }
 
-dimension: num_search_engine_impressions {
-  hidden:  yes
-}
+  dimension: sponsored_suggestions_enabled {
+    sql: ${TABLE}.pref_sponsored_suggestions ;;
+    type: yesno
+    group_label: "User Preferences"
+    description: "Whether or not the checkbox for showing sponsored suggestions is checked in about:preferences"
+  }
 
-dimension: num_total_results {
-  hidden:  yes
-}
+  dimension: product_engaged_result_type {
+    hidden:  yes
+  }
 
-dimension: num_trending_suggestion_impressions {
-  hidden:  yes
-}
+  dimension: product_first_result_type {
+    sql: ARRAY(SELECT r.product_result_type FROM UNNEST(${results}) AS r WHERE r.position = 1)[SAFE_OFFSET(0)] ;;
+    type: string
+  }
 
-dimension: num_weather_impressions {
-  hidden:  yes
-}
+  dimension: product_selected_result {
+    hidden:  yes
+  }
 
-dimension: num_wikipedia_dynamic_impressions {
-  hidden:  yes
-}
+  dimension: sample_id {
+    group_label: "Filters to speed up Looker"
+    description: "Filter on this Dimension to speed up Looker while prototyping a dashboard. For example, filtering `sample_id < 10` will select a random 10% sample of the data, instead of all the data. DO NOT use this filter in a production dashboard for metrics with rare events (e.g., when click counts on a result type are low)."
+  }
 
-dimension: num_wikipedia_enhanced_impressions {
-  hidden:  yes
-}
+  dimension: selected_position {
+    hidden:  yes
+  }
 
-dimension: product_engaged_result_type {
-  hidden:  yes
-}
+  dimension: selected_result {
+    hidden:  yes
+  }
 
-dimension: product_first_result_type {
-  hidden:  yes
-}
+  dimension: seq {
+    hidden:  yes
+  }
 
-dimension: sample_id {
-  group_label: "Filters to speed up Looker"
-  description: "Filter on this Dimension to speed up Looker while prototyping a dashboard. For example, filtering `sample_id < 10` will select a random 10% sample of the data, instead of all the data. DO NOT use this filter in a production dashboard for metrics with rare events (e.g., when click counts on a result type are low)."
-}
+  dimension_group: submission {
+    sql: ${TABLE}.submission_date ;;
+    type: time
+    timeframes: [
+      raw,
+      date,
+      week,
+      month,
+      quarter,
+      year,
+    ]
+    convert_tz: no
+    datatype: date
+  }
 
-dimension: session_action_type {
-  group_label: "Urlbar-specific filters"
-  description: "The type of urlbar event: engagement, abandonment or annoyance."
-}
+  dimension: num_autofill_impressions {
+    hidden:  yes
+    sql:  ARRAY_LENGTH(ARRAY(SELECT 1 FROM UNNEST(results) AS r WHERE r.product_result_type = "autofill"));;
```

Comment:
```suggestion
    sql:  (select countif(r.product_result_type = "autofill") from unnest(results) as r);;
```

Reply:
lol yes, one day I will learn to use Countif!